cluster/images/conformance/Makefile

```diff
@@ -31,7 +31,7 @@ E2E_GO_RUNNER_BIN?=$(shell test -f $(LOCAL_OUTPUT_PATH)/go-runner && echo $(LOCA
 
 CLUSTER_DIR?=$(shell pwd)/../../../cluster/
 
-BASEIMAGE=debian:stable-slim
+BASEIMAGE=gcr.io/distroless/base:latest
```

Comment:
I guess we don't have this in k8s.gcr.io?
Xref: https://github.com/kubernetes/kubernetes/issues/99238

Reply:
@BenTheElder sorry, re-reading this -- did you mean to imply that we should add this to k8s.gcr.io?

Reply:
something to consider in a follow-up (or using go-runner?)

Reply:
+1 to using go-runner here instead!

Reply:
my initial attempt using go-runner didn't work, but i'll try again in a follow up!